### PR TITLE
examples: update examples with-ant-design

### PR DESCRIPTION
**Pull request by li-jia-nan**
demo: update examples with-ant-design

- https://github.com/ant-design/ant-design/pull/43663

### CODE REVIEW

**Comment by li-jia-nan** [comment]
@huozhi @timneutkens @styfle Hi, can you review this PR for me ? thx~

**Comment by li-jia-nan** [comment]
image

File: examples/with-ant-design/pages/index.tsx [hunk 1]
```diff
@@ -7,31 +7,27 @@ import {
...
-  ) => {
-    console.log(date, dateString)
-  }
-
+function Home() {
   return (
-    <div style={content}>
+    <div
+      className={inter.className}
+      style={{ padding: '100px 0', height: '100vh' }}
```
Comment (li-jia-nan) [inline examples/with-ant-design/pages/index.tsx:25]:
```suggestion
      style={{ padding: 100, height: '100vh' }}
```

**Comment by li-jia-nan** [comment]
@huozhi @timneutkens @styfle @timeyoutakeit @Nutlope  Hello, I'm li-jia-nan from the ant-desgin team. The change of this PR is to make the example more reasonable. Can someone review it for me? thank you very much~

File: examples/with-ant-design/pages/_document.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <Head />
+      <body>
+        <Main />
+        <NextScript />
+      </body>
+    </Html>
+  )
+}
+
+MyDocument.getInitialProps = async (ctx: DocumentContext) => {
```
Comment (leerob) [inline examples/with-ant-design/pages/_document.tsx:17]: We do not recommend this approach. 

https://nextjs.org/docs/pages/api-reference/functions/get-initial-props

CleanShot 2023-07-24 at 18 00 46@2x

**Review by leerob — CHANGES_REQUESTED** [review]

File: examples/with-ant-design/pages/_document.tsx (hunk 2 above)
Comment (li-jia-nan) [reply examples/with-ant-design/pages/_document.tsx:17]: Bug `_document` can only use `getInitialProps`, and `getStaticProps` can only be exported from a `page`

Can you give me some hints? About the correct way of writing here, thx

Comment (leerob) [reply examples/with-ant-design/pages/_document.tsx:17]: If you need layout level data fetching, then I would recommend exploring the App Router.

Comment (li-jia-nan) [reply examples/with-ant-design/pages/_document.tsx]: Hello, I have changed to app router mode, pls review again, thx :)

File: examples/with-ant-design/app/api/hello.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
```
Comment (leerob) [inline examples/with-ant-design/app/api/hello.ts:2]: Let's remove this file

File: examples/with-ant-design/app/globals.css [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  --background-start-rgb: 214, 219, 220;
+  --background-end-rgb: 255, 255, 255;
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --foreground-rgb: 255, 255, 255;
+    --background-start-rgb: 0, 0, 0;
+    --background-end-rgb: 0, 0, 0;
+  }
```
Comment (leerob) [inline examples/with-ant-design/app/globals.css:16]: Are these being used?

File: examples/with-ant-design/app/api/hello.ts (hunk 3 above)
Comment (li-jia-nan) [reply examples/with-ant-design/app/api/hello.ts]: I have removed  this file

File: examples/with-ant-design/app/globals.css (hunk 4 above)
Comment (li-jia-nan) [reply examples/with-ant-design/app/globals.css]: Thanks for the reminder, I have removed these

**Review by leerob — APPROVED** [review]

**Pull request merged**